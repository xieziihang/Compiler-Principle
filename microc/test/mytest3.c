void main() {
  int i;
  i = 1;
  i == 2? i = 5: i = 2;
  print i;
}