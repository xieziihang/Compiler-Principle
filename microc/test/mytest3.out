24 19 0 5 25 15 1 13 0 0 1 0 2 12 15 -1 13 0 0 1 11 0 2 6 17 35 13 0 0 1 0 5 12 16 42 13 0 0 1 0 2 12 15 -1 13 0 0 1 11 22 15 -1 15 -1 21 -1